pkg/blobplugin/Dockerfile

```diff
@@ -29,6 +29,8 @@ RUN chmod +x /blobfuse-proxy/init.sh && \
  chmod +x /blobfuse-proxy/blobfuse-proxy
 
 RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase
+RUN wget -c https://azcopyvnext.azureedge.net/release20230420/azcopy_linux_amd64_10.18.1.tar.gz && tar -zxvf azcopy_linux_amd64_10.18.1.tar.gz && \
+  mv ./azcopy_linux_amd64_10.18.1/azcopy azcopy && rm -rf azcopy_linux_amd64_10.18.1 && rm -rf azcopy_linux_amd64_10.18.1.tar.gz
```

Comment:
remove first space, and where is `azcopy`, it's under `/` root folder, pls make it explicitly

Reply:
done